### ISSUE TITLE
Index was outside of the bounds of the array error.

### ISSUE THREAD

**Issue by wil-sys** [AssetRipper/AssetRipper#228]
[AssetRipper.log](https://github.com/AssetRipper/AssetRipper/files/7841914/AssetRipper.log)
I was told from Reddit to make an issue here. The game I'm trying to decompile is Virtual-Virtual Reality.

**Comment by SamboyCoding** [comment]
Ideally, you would have filed your issue using the actual bug report template so we have more information about your environment, but the log will suffice.

**Comment by wil-sys** [comment]
> Ideally, you would have filed your issue using the actual bug report template so we have more information about your environment 
Oh, sorry I did not now there was a template, I could rewrite my issue using the template if that would make it easer.

**Comment by ds5678** [comment]
> > Ideally, you would have filed your issue using the actual bug report template so we have more information about your environment 
> 
> Oh, sorry I did not now there was a template, I could rewrite my issue using the template if that would make it easer.
> 
> 

It's not necessary to post a new issue. Just upload your AssetRipper log.

**Comment by ds5678** [comment]
This is a read error. A pptr has the wrong offset in the monobehaviour structure causing it to look in the wrong place for that dependency. I've posted #229 to better describe this issue. Implementation is not likely until after #164. 

As a work-around, you can disable script export.
